app-ng/src/config.lisp

```diff
@@ -0,0 +1,129 @@
+(in-package #:qvm-app-ng)
+
+(global-vars:define-global-var *config*
```

Comment:
By convention I think these `global-vars` usually have an extra pair of earmuffs: `**config**`